### PR TITLE
bump-periodics.sh: Prefer minimum_interval over interval

### PR DESCRIPTION
**Pull request by mandre**
We want the jobs to use the newly introduced periodic type to spread them better.

### CODE REVIEW

File: bump-periodics.sh [hunk 1]
```diff
@@ -65,7 +65,7 @@ yq --yaml-output --in-place '.
 	| ( .releases[][] | select(.stream == "ci") | .stream ) |= "nightly"
 
 	# Set a conveniently long interval to all tests in the old periodics
-	| .tests[].interval |= "72h"
+	| .tests[].minimum_interval |= "72h"
```
Comment (pierreprinetti) [inline bump-periodics.sh:68]: This change assumes that the OLD_VERSION job has `minumum_interval`. We should probably first remove any {interval,cron} property before adding our `minimum_interval`, to make the script more robust.

Something like:
```plaintext
| del(.tests[].interval)
| del(.tests[].cron)
| .tests[].minimum_interval |= "72h"
```

Comment (mandre) [reply bump-periodics.sh:70]: Good idea :+1:

File: bump-periodics.sh [hunk 2]
```diff
@@ -10,12 +10,14 @@ PERIODICS_DIR="${2}/ci-operator/config/shiftstack/shiftstack-ci"
 
 OLD_VERSION="4.$(( ${NEW_VERSION#4.} - 1 ))"
 OLD_OLD_VERSION="4.$(( ${OLD_VERSION#4.} - 1 ))"
+OUT_OF_SUPPORT_VERSION="4.$(( ${OLD_VERSION#4.} - 5 ))"
 
-OLD_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${OLD_VERSION}.yaml" 
-NEW_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${NEW_VERSION}.yaml" 
+OLD_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${OLD_VERSION}.yaml"
+NEW_PERIODIC="${PERIODICS_DIR}/shiftstack-shiftstack-ci-main__periodic-${NEW_VERSION}.yaml"
```
Comment (pierreprinetti) [inline bump-periodics.sh:16]: Am I blind? What changed in these two lines?

Comment (mandre) [reply bump-periodics.sh:16]: Trimmed empty chars at the end of the line.

Comment (pierreprinetti) [reply bump-periodics.sh:16]: weird. Github usually highlights those (like it did in the following lines)

**Pull request merged**